src/software/jetson_nano/redis/redis_client.cpp

```diff
@@ -45,16 +45,50 @@ RedisClient::RedisClient(std::string host, size_t port)
     subscriber_.commit();
 }
 
-std::string RedisClient::get(const std::string &key)
+std::string RedisClient::getSync(const std::string &key)
 {
     auto future = client_.get(key);
     client_.commit();
     return future.get().as_string();
 }
 
-void RedisClient::set(const std::string &key, const std::string &value)
+void RedisClient::getAsyncNoCommit(const std::string &key,
```

Comment:
do we need this function? With no commit the caller is relying on other functions to be called that use commit. Might become buggy with the callback in the future.

Reply:
I guess what I wanted to do was reduce the amount of "network transfers" we have to make to the redis server (even though its running on the nano) I figured having more control over how we commit will let us send multiple in one packet